_posts/2022-12-19-changelog-185.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #185"
+date: 2022-12-19 13:00:00 +0300
+---
+
+## New Features
+
+* [#9803] Show [type hints](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-reference-info.html#inlay-hints) inside attribute procedural macros
```

Comment:
Let's mention that attribute macros are not enabled by default for now and not to turn them on
Also, I'd swap this item with type inference for GATs since it's an improvement of experimental feature => affects small number of users